src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -110,6 +112,33 @@ const SubmissionDetails = _.flow(
   )(workflows)
 
   const { succeeded, failed, running, submitted } = _.groupBy(wf => collapseStatus(wf.status), workflows)
+
+  // Note: This 'deleteLimitYears' value should reflect the current 'deletion-delay' value configured for PROD in firecloud-develop's
+  // 'cromwell.conf.ctmpl' file:
+  const deletionDelayYears = 6
+  const deletionDelayString = `${deletionDelayYears} year${deletionDelayYears > 1 ? 's' : ''}`
+  const isDeleted = statusLastChangedDate => differenceInDays(Date.parse(statusLastChangedDate), Date.now()) > (deletionDelayYears * 365)
+
+  const deletedInfoIcon = ({ name, iconOverride }) => {
+    return h(InfoBox, {
+      style: { color: colors.secondary(), margin: '0.5rem' },
+      tooltip: `${name} unavailable. Click to learn more.`,
+      iconOverride
+    }, [
+      div({ style: Style.elements.sectionHeader }, 'Workflow Details Archived'),
+      div({ style: { padding: '0.5rem 0' } }, [`This workflow's details have been archived (> ${deletionDelayString} old).`]),
```

Comment:
TOL, I think it flows a bit smoother without the parenthetical and symbolic greater-than
```suggestion
      div({ style: { padding: '0.5rem 0' } }, [`This workflow ran more than ${deletionDelayString} ago and is archived.`]),
```
<img width="322" alt="Screen Shot 2021-05-13 at 12 54 32 PM" src="https://user-images.githubusercontent.com/1087943/118159012-829d5400-b3ea-11eb-825f-393773147561.png">